extensions/spark/install_spark.sh

```diff
@@ -128,7 +129,6 @@ export SPARK_DAEMON_MEMORY=${SPARK_DAEMON_MEMORY}m
 export SPARK_WORKER_DIR=${SPARK_WORKDIR}
 export SPARK_LOCAL_DIRS=${SPARK_TMPDIR}
 export SPARK_LOG_DIR=${SPARK_LOG_DIR}
-export SPARK_DIST_CLASSPATH+=:\$(${HADOOP_INSTALL_DIR}/bin/hadoop classpath)
```

Comment:
@dennishuo: It looks like this is including a whole bunch of hadoop dependencies.  A lot of those dependencies are extremely old... for example, hadoop has guava-11.0.2.jar all over the place and hbase had guava-12.0.1.jar.  those versions are both old, and likely incompatible.  By default, hbase includes hadoop 2.5.1 instead of 2.6.

Is there a way to reduce the scope of the SPARK_DIST_CLASSPATH additions so as not to hit these incompatibilities?  I'm assuming that it won't just be us that has this type of dependency class due to the addition of $(hadoop classpath)